src/kvstore/Common.h

```diff
@@ -14,20 +14,40 @@
 namespace nebula {
 namespace kvstore {
 
+#define X_ERROR \
```

Comment:
Implement errMsg directly.  Don't write code in this way.  Thank you

Reply:
Any problem?

Reply:
For human-readable,  if not necessary,  don't use MACRO in our code base.

Reply:
Yes, I think it's valuable to avoid duplicate definition with some abstraction.